src/DynamoAnalyzer/Helper/CsvHandler.cs

```diff
@@ -0,0 +1,151 @@
+using System.Globalization;
+using CsvHelper;
+using CsvHelper.Configuration;
+using CsvHelper.TypeConversion;
+using DynamoAnalyzer.Models;
+using Newtonsoft.Json;
+
+namespace DynamoAnalyzer.Helper
+{
+    /// <summary>
+    /// Provides methods to work with csv files
+    /// </summary>
+    public static class CsvHandler
```

Comment:
in general, make everything internal unless theres a good reason to make it public.

Reply:
Got it